README.md

```diff
@@ -39,20 +39,29 @@ Install it with:
 
     cabal update && cabal install --ignore-project doctest
 
-Make sure that Cabal's `bindir` is on your `PATH`.
+Make sure that Cabal's `installdir` is on your `PATH`.
 
-On Linux:
+On Linux / macOS / BSD:
 
-    export PATH="$(cabal -v0 path --installdir):$PATH"
-
-On Mac OS X:
-
-    export PATH="$(cabal -v0 path --installdir):$PATH"
+```bash
+ # requires cabal-install 1.12, or later
+export PATH="$(cabal -v0 path --installdir):$PATH"
+```
+or
+```bash
+export PATH="$HOME/.local/bin:$PATH"
+```
 
 On Windows:
 
-    set PATH="%AppData%\cabal\bin\;%PATH%"
-
+```batch
+: requires cabal-install 1.12, or later
+for /f %i in ('cabal path -v0 --installdir') do set PATH=%i;%PATH%
+```
+or
+```batch
+set PATH="%AppData%\cabal\bin\;%PATH%"
```

Comment:
@mpilgrem do you know if this is still the default location on windows? The Cabal docs only say `~/.local/bin`.
https://cabal.readthedocs.io/en/stable/config.html#directories

Reply:
The short answer is "Yes, but" - See `Distribution.Client.Config.defaultInstallPath` and `maybeGetCabalDir` of the `cabal-install` package; the latter uses `getAppUserDataDirectory "cabal"`.

The "but" is that I think the concept of 'default' depends on how the user manages versions of Cabal (the tool). If they use GHCup to do so (as I do), then `cabal path --installdir` yields `C:\cabal\bin`.

Reply:
Again, thanks a lot @mpilgrem.

> If they use GHCup to do so (as I do), then `cabal path --installdir` yields `C:\cabal\bin`.

OK, that makes sense. That's consistent with what I am seeing on GitHub actions.

I am going to only document the cabal 3.12 way for windows then. Admittedly that's not perfect currently, but as time goes by that will work for the majority of the users.